edgebuilder-install.sh

```diff
@@ -384,7 +389,7 @@ if [ "$COMPONENT" = "server" ];then
 elif [ "$COMPONENT" = "node" ]; then
 
   if [ "$ARCH" = "x86_64" ]||[ "$ARCH" = "aarch64" ];then
-    if [ "$OS" = "$UBUNTU2004" ]||[ "$OS" = "$UBUNTU1804" ]||[ "$OS" = "$DEBIAN10" ];then
+    if [ "$OS" = "$UBUNTU2104" ] || [ "$OS" = "$UBUNTU2004" ]||[ "$OS" = "$UBUNTU1804" ]||[ "$OS" = "$DEBIAN10" ];then
```

Comment:
That's not right? We currently dont support node components, do we?

Reply:
No i forgot to change that. Changes have been made now